New features (#8)
reportportal/agent-python-behave: README.rst

- :code:`launch_name = AnyLaunchName` - launch name (default value is 'Python Behave Launch')
- :code:`launch_id = xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` - id of the existing launch (the session will not handle the lifecycle of the given launch)
- :code:`launch_attributes = Smoke Env:Python3` - list of attributes for launch
- :code:`launch_description = Smoke test` - launch description
- :code:`step_based = True` - responsible for Scenario or Step based logging (Scenario based approach is used by default)

> - :code:`is_skipped_an_issue = False` - if it True, skipped tests are not marked as issue

Reviewer: ```suggestion
- :code:`is_skipped_an_issue = False` - a flag to skip tests which are not marked as an issue
```

Author: The idea is to provide ability not to mark skipped tests as Issue in RP. they are not marked with some 'issue' attribute in code

Reviewer: It's not about the content but about spelling.

Author: From my point of view it changes the meaning of the comment. I can suggest comment from client "option to mark skipped tests as not 'To Investigate' items on Server side."

Author: done
